src/integration-test/kotlin/integration/helper/IntegrationHelperUtil.kt

```diff
@@ -69,40 +75,64 @@ class IntegrationHelperUtil {
     val tokenContracts = mutableMapOf<String, BasicCoin>()
 
     /** Provider that is used to store/fetch tokens*/
-    private val ethTokensProvider = EthTokensProviderImpl(
-        testConfig.iroha,
-        irohaKeyPair,
-        testConfig.notaryIrohaAccount,
-        tokenStorageAccount
-    )
+    private val ethTokensProvider by lazy {
+        EthTokensProviderImpl(
+            testConfig.iroha,
+            irohaKeyPair,
+            testConfig.notaryIrohaAccount,
+            tokenStorageAccount
+        )
+    }
 
     private val relayRegistrationConfig =
         loadConfigs("test", RelayRegistrationConfig::class.java, "/test.properties")
 
     /** Provider that is used to get free registered relays*/
-    private val ethFreeRelayProvider = EthFreeRelayProvider(
-        testConfig.iroha,
-        irohaKeyPair,
-        testConfig.notaryIrohaAccount,
-        registrationAccount
-    )
+    private val ethFreeRelayProvider by lazy {
+        EthFreeRelayProvider(
+            testConfig.iroha,
+            irohaKeyPair,
+            testConfig.notaryIrohaAccount,
+            registrationAccount
+        )
+    }
 
     /** Provider of ETH wallets created by registrationAccount*/
-    private val ethRelayProvider = EthRelayProviderIrohaImpl(
-        testConfig.iroha, irohaKeyPair, testConfig.notaryIrohaAccount, registrationAccount
-    )
+    private val ethRelayProvider by lazy {
+        EthRelayProviderIrohaImpl(
+            testConfig.iroha, irohaKeyPair, testConfig.notaryIrohaAccount, registrationAccount
+        )
+    }
 
-    /** Provider of ETH wallets created by registrationAccount*/
-    private val registrationStrategy =
-        RegistrationStrategyImpl(
+    private val ethRegistrationStrategy by lazy {
+        EthRegistrationStrategyImpl(
             ethFreeRelayProvider,
             irohaConsumer,
             testConfig.notaryIrohaAccount,
             registrationAccount
         )
+    }
+
+    private val btcRegistrationStrategy by lazy {
+        BtcRegistrationStrategyImpl(
+            irohaConsumer,
+            testConfig.notaryIrohaAccount,
+            btcRegistrationAccount,
+            testConfig.bitcoin.walletPath
+        )
+    }
 
     private val relayRegistration = RelayRegistration(relayRegistrationConfig, passwordConfig)
 
+    fun registerBtcAddress(irohaAccountName: String): String {
+        val keypair = ModelCrypto().generateKeypair()
+        btcRegistrationStrategy.register(irohaAccountName, keypair.publicKey().hex())
+            .fold({ btcAddress ->
+                logger.info { "newly registered btc address $btcAddress" }
+                return btcAddress
+            }, { ex -> throw ex })
```

Comment:
I think it is better to return ` Result<String, Exception>`

Reply:
I believe this is not relevant, because it's just a simple helper class that is used in tests only